base_rotation_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: base_rotation_joint
    joint_speed: 0.1
    max_torque: 0.9
    motor:
        id: 1
        init: 2048
        min: 1025
        max: 3072

shoulder_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller_dual_motor
        type: JointPositionControllerDual
    joint_name: shoulder_joint
    joint_speed: 0.1
    max_torque: 0.9
    motor_master:
        id: 2
        init: 2048
        min: 947
        max: 3158
    motor_slave:
        id: 3

elbow1_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: elbow1_joint
    joint_speed: 0.3
    max_torque: 0.9
    motor:
        id: 4
        init: 2048
        min: 457
        max: 3640

elbow2_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: elbow2_joint
    joint_speed: 0.3
    max_torque: 0.9
    motor:
        id: 5
        init: 2048
        min: 457
        max: 3640

wrist_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: wrist_joint
    joint_speed: 0.3
    max_torque: 0.9
    motor:
        id: 6
        init: 2048
        min: 798
        max: 3299

right_finger_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: right_finger_joint
    joint_speed: 0.2
    max_torque: 0.1
    motor:
        id: 7
        init: 2048
        min: 1480
        max: 2622

left_finger_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: left_finger_joint
    joint_speed: 0.2
    max_torque: 0.1
    motor:
        id: 8
        init: 2048
        min: 1480
        max: 2622
elevator_controller:
    controller:
        package: dynamixel_controllers
        module: joint_torque_controller
        type: JointTorqueController
    joint_name: elevator_joint
    joint_speed: 0.0
    joint_torque_limit: 0.6
    motor:
        id: 9
        init: 2048
        min: 0
        max: 4095

